docs/docusaurus/docs/changelog.md

```diff
@@ -10,6 +10,21 @@ title: Changelog
 - Deprecation warnings are accompanied by a moniker (as a code comment) indicating when they were deprecated.  For example: `# deprecated-v0.13`
 - Changes to methods and parameters due to deprecation are also noted in the relevant docstrings.
 
+### 0.18.16
+* [FEATURE] SnowflakeDatasource - Require `database` and `schema` fields ([#9980](https://github.com/great-expectations/great_expectations/pull/9980))
+* [FEATURE] Snowflake - use `Datasource` level `schema` when creating `TableAsset` ([#9992](https://github.com/great-expectations/great_expectations/pull/9992))
```

Comment:
Both of these changes were reverted in a subsequent PR.
Should we even include this in the changelog.

My instinct is to say remove them... but there are some small QOL updates in these PR's that were retained even if the titular "features" were reverted. 🤷

Reply:
I would remove these and the revert PRs from the changelog. Or prefix `REVERTED -` to the description.